ui/raidboss/data/05-shb/raid/e8s.js

```diff
@@ -705,294 +705,295 @@
-        'Wyrmfang': 'Crocs du Dragon divin',
-        'Wyrmclaw': 'Griffes du Dragon divin',
-        'Thin Ice': 'Verglas',
-        'Refulgent Fate': 'Lien de Lumière',
-        'Refulgent Chain': 'Chaînes de Lumière',
-        'Physical Vulnerability Up': 'Vulnérabilité physique augmentée',
-        'Magic Vulnerability Up': 'Vulnérabilité magique augmentée',
-        'Lightsteeped': 'Lumière excédentaire',
-        'Light Resistance Down': 'Résistance à la Lumière réduite',
-        'Heavy': 'Pesanteur',
-        'Hated of the Wyrm': 'Malédiction du Dragon divin',
-        'Hated of Frost': 'Malédiction de la Furie des neiges',
-        'Freezing': 'Congélation graduelle',
-        'Down for the Count': 'Au tapis',
-        'Deep Freeze': 'Congélation',
         'Damage Down': 'Malus de dégâts',
+        'Deep Freeze': 'Congélation',
+        'Down for the Count': 'Au tapis',
+        'Freezing': 'Congélation graduelle',
+        'Hated of Frost': 'Malédiction de la Furie des neiges',
+        'Hated of the Wyrm': 'Malédiction du Dragon divin',
+        'Heavy': 'Pesanteur',
+        'Light Resistance Down': 'Résistance à la Lumière réduite',
+        'Lightsteeped': 'Lumière excédentaire',
+        'Magic Vulnerability Up': 'Vulnérabilité magique augmentée',
+        'Physical Vulnerability Up': 'Vulnérabilité physique augmentée',
+        'Refulgent Chain': 'Chaînes de Lumière',
+        'Refulgent Fate': 'Lien de Lumière',
+        'Thin Ice': 'Verglas',
+        'Wyrmclaw': 'Griffes du Dragon divin',
+        'Wyrmfang': 'Crocs du Dragon divin',
       },
     },
     {
       'locale': 'ja',
       'missingTranslations': true,
       'replaceSync': {
-        'luminous Aether': 'ライト・エーテル',
-        'holy light': '聖なる光',
-        'great wyrm': '聖竜',
-        'frozen mirror': '氷面鏡',
-        'electric Aether': 'ライトニング・エーテル',
-        'earthen Aether': 'アース・エーテル',
         'aqueous Aether': 'ウォーター・エーテル',
-        'Shiva': 'シヴァ',
+        'earthen Aether': 'アース・エーテル',
+        'electric Aether': 'ライトニング・エーテル',
+        'frozen mirror': '氷面鏡',
+        'great wyrm': '聖竜',
+        'holy light': '聖なる光',
+        'luminous Aether': 'ライト・エーテル',
         'Mothercrystal': 'マザークリスタル',
+        'Shiva': 'シヴァ',
       },
       'replaceText': {
+        'attack': '攻撃',
+        'Absolute Zero': '絶対零度',
+        'Akh Morn': 'アク・モーン',
+        'Akh Rhai': 'アク・ラーイ',
+        'Axe/Scythe Kick': 'アクスキック/サイスキック',
+        'Banish(?! )': 'バニシュ',
+        'Banish Divided': 'ディバイデッド・バニシュ',
+        'Banish III': 'バニシュガ',
+        'Banish III Divided': 'ディバイデッド・バニシュガ',
+        'Biting/Driving Frost': 'フロストスラッシュ/フロストスラスト',
+        'Bright Hunger': '浸食光',
+        'Bright Pulse': '閃光',
+        'Diamond Frost': 'ダイヤモンドダスト',
+        'Double Slap': 'ダブルスラップ',
+        'Drachen Armor': 'ドラゴンアーマー',
+        'Draconic Strike': 'ドラコニックストライク',
+        'Driving/Biting Frost': 'フロストスラスト/フロストスラッシュ',
+        'Embittered/Spiteful Dance': 'Embittered/冷厳の舞踏技',
```

Comment:
Need help for fix Embittered plz. @maiko-tan-coding  @Jaehyuk-Lee

Reply:
`'Embittered/Spiteful Dance': '峻厳の舞踏技／冷厳の舞踏技',`

I recommend using full-width slash `／` here in ja translation, instead of the half-width one `/`, to match other text.
Additionally, I have not entered the savage raid in this patch ( stay in Ishgard Firmament every time... ), so I don't know if it is correct. These words are extracted from game data.